### ISSUE TITLE
Engagement Change History

### ISSUE THREAD

**Issue by tiagograf-AOT** [bcgov/met-public#249]

**Comment by marieanik** [comment]
@sidtobias I consider this to be really low priority. The one use case would be if someone override someone else changes to revert back one version but once the engagement goes to an admin and is approved, there shouldn't be anymore changes other than fixing a misspelling so there will always be only one version. All the rest would be drafts.

**Comment by sidtobias** [comment]
Agree however they will be required just not as a priority

Sid Tobias
Director, Digital Services
Environmental Assessment Office
Government of British Columbia
Mobile:  (250) 893-6218
[signature_536649950]Twitter.com/BC_EAO<http://twitter.com/BC_EAO>

***@***.***
